grouper/settings.py

```diff
@@ -79,6 +79,7 @@ def __init__(self):
         self.http_proxy_host = None  # type: Optional[str]
         self.http_proxy_port = None  # type: Optional[int]
         self.log_format = "%(asctime)-15s\t%(levelname)s\t%(message)s  [%(name)s]"
+        self.log_redact_patterns = [] # type: List[str]
```

Comment:
This seems unused?

Reply:
Yeah. Leftover from when I had it in the config/settings files. I'll remove it.